tools/alpine/detect-arch-var

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env sh
```

Comment:
hmm, yes I filed an issue https://github.com/moby/moby/issues/34199 will try to get it fixed ASAP.

As a temporary workaround, you could omit setting `DOCKER_CONTENT_TRUST` on the build step if the build arg is set.